### PR TITLE
Props Table:  Add description to  types of all components

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

File: packages/components/src/Table/Table.tsx [hunk 1]
```diff
@@ -36,8 +36,23 @@ export interface TableProps
...
+
+  /**
+   * Stretch table to fill the container.
+   * @default false
+   */
   stretch?: boolean;
+
+  /**
+   * Make the column sticky to the top of the table.
+   * @default none
```
Comment (sarahgm) [inline packages/components/src/Table/Table.tsx:48]: i think this should be default false right?

File: packages/components/src/Tiles/Tiles.tsx [hunk 2]
```diff
@@ -3,25 +3,43 @@ import { ReactNode } from 'react';
...
+   */
   equalHeight?: boolean;
 }
 
 export const Tiles = ({
   space = 0,
   stretch = false,
   equalHeight = false,
-  tilesWidth,
+  tilesWidth = '250px',
```
Comment (sarahgm) [inline packages/components/src/Tiles/Tiles.tsx:34]: why is there a default value?

File: packages/components/src/Tooltip/Tooltip.tsx [hunk 3]
```diff
@@ -10,9 +10,15 @@ import { TooltipTrigger } from './TooltipTrigger';
 type RemovedProps = 'className' | 'isOpen' | 'style';
 
 export interface TooltipProps extends Omit<RAC.TooltipProps, RemovedProps> {
+  /**
+   * The children of the coomponent.
```
Comment (sarahgm) [inline packages/components/src/Tooltip/Tooltip.tsx:14]: typo by component

File: packages/components/src/Tiles/Tiles.tsx (hunk 2 above)
Comment (OsamaAbdellateef) [reply packages/components/src/Tiles/Tiles.tsx:34]: oh, i think i was testing & forgot it :D @sarahgm

Comment (OsamaAbdellateef) [reply packages/components/src/Tiles/Tiles.tsx:34]: but i noticed it exists our docs table with default value `250px` idk why @sarahgm

File: packages/components/src/Table/Table.tsx (hunk 1 above)
Comment (OsamaAbdellateef) [reply packages/components/src/Table/Table.tsx]: yeah it shouldn't be none, but i think it should be true

**Review by sarahgm — APPROVED** [review]

**Pull request merged**